pom.xml

```diff
@@ -249,25 +249,26 @@
         <artifactId>mockito-all</artifactId>
         <version>1.9.0</version>
       </dependency>
-      <dependency>
-        <groupId>javax.jdo</groupId>
```

Comment:
Why were these jdo changes made? These Maven files wouldn't really have been modified internally and I don't see any Java code modifications that would trigger this change.